test/scripts/openshift-ci/install-operator.sh

```
@@ -0,0 +1,413 @@
+            echo \"  CSV phase: \${phase}\"
+            if [[ \"\${phase}\" == \"Succeeded\" ]]; then
+                break
+            fi
+            sleep 10
+        done
+    "
+    echo "${OPERATOR_NAME} installed successfully"
+
+    echo "Waiting for ${CONTROLLER_DEPLOYMENT} deployment to be available..."
+    timeout 300 bash -c "
+        while ! oc get deployment ${CONTROLLER_DEPLOYMENT} -n "${OPERATOR_NAMESPACE}" &>/dev/null; do
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)